bokeh/models/tools.py

```diff
@@ -392,6 +392,41 @@ class BoxZoomTool(Drag):
 
     """)
 
+class ZoomInTool(Action):
+    """ *toolbar icon*: |zoom_in_icon|
+
+    The zoom-in tool allows users to click a button to zoom in
+    by a fixed amount.
+
+    .. |zoom_in_icon| image:: /_images/icons/ZoomIn.png
+        :height: 18pt
+
+    """
+    # TODO ZoomInTool dimensions should probably be constrained to be the same as ZoomOutTool
```

Comment:
I'd remove the TODO for now. Future work around bundling tools together should handle coalescing dimensions or other related properties, but we'll handle it then.